Makefile

```diff
@@ -12,4 +12,14 @@ test:
 	go test ./...
 
 proto_gen:
-	protoc -I grpc/pb/ grpc/pb/space.proto --go_out=plugins=grpc:grpc/pb
\ No newline at end of file
+	protoc -I grpc/pb/ -I grpc/proto/ -I${GOPATH}/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis grpc/proto/space.proto --go_out=plugins=grpc:grpc/pb
+
+gen_proxy:
```

Comment:
Maybe we call this `gen_gateway` or `gen_rest` since it might get confused with the web proxy.

Reply:
Fixed